### PR TITLE
Fix patients UI

### PR DESCRIPTION
**Pull request by jadh4v**
Addresses: https://github.com/KitwareMedical/lungair-web-application/issues/23

- Improved ordering of tabs in the ModulePanel (Patients, Images, annotations, Analysis)
- Better tab icons for ModulePanel
- UI for patients list: scrollable list, selected patient highlighted
- Cleaner Image list interface: reduced collapsible panels in list of images

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -22,4 +22,5 @@ VITE_LOCAL_FHIR_SERVER_URL=http://localhost:3000/hapi-fhir-jpaserver/fhir
 # Display name of the Local FHIR host.
 VITE_LOCAL_FHIR_SERVER_NAME=Kitware FHIR SandBox
 
-VITE_PATIENT_IDENTIFIER="ID assigned as part of synthetic data generation"
+# VITE_PATIENT_IDENTIFIER="ID assigned as part of synthetic data generation"
+VITE_PATIENT_IDENTIFIER="ID column from original data excel spreadsheet"
```
Comment (ebrahimebrahim) [inline .env:26]: let's not commit this change just so the default settings don't require having the spreadsheet

as part of #23 we also said we'd explain what is `VITE_PATIENT_IDENTIFIER` in the README. I can add that later if you want or you can add -- let's just not forget it

**Review by ebrahimebrahim — COMMENTED** [review]
I like the chart and lung icons :)

The patient list looks great

I found one bug while testing: selecting a patient, expanding their image panel, and then trying to go back and select a different patient breaks the application. Here is a gif showing what I mean:

Peek 2023-12-19 10-43

File: .env (hunk 1 above)
Comment (jadh4v) [reply .env]: @ebrahimebrahim I have updated the PR with the local fixes we had discussed last time. Also updated the README as you have mentioned here.
Please take a second look at the README updates, then I will merge.

There are some additional bugs:
```
	- Cannot re-click image after loading once.
	- Image List not scrollable
```
But these are related to the ImageList and not the PatientList, so I was wondering if we should do it as a different PR.
I haven't looked into this yet, but if the fixes are simple, I will just push additional commits on this PR.

**Comment by ebrahimebrahim** [comment]
Tested and it works great.

The things I mentioned in the last review are all addressed, thanks

There are just the items you listed:
- Cannot re-click image after loading once.
- Image List not scrollable

and there is an additional bug I just noticed:
- Only the first page of patients from the fhir patients query is actually loaded. All other patients are never loaded. The output of the fhir query is actually paginated and you have to deal with the "next" links that it contains to retrieve all patients. There is a way to increase the maximum number of patients that get included in one query, so a first step would be to increase that number. But even that maximum can only be so large and at some point one has to deal with the pagination.

Let's go ahead and merge this PR and I will open bug-issues for those remaining items. Thanks for all your work on this!

**Pull request merged**